src/server_name.rs

```diff
@@ -49,6 +51,18 @@ impl<'a> ServerName<'a> {
             Self::IpAddress(i) => ServerName::IpAddress(*i),
         }
     }
+
+    /// Return the string representation of this `ServerName`.
+    #[cfg(feature = "std")]
+    pub fn to_str(&self) -> Cow<'_, str> {
+        match self {
+            Self::DnsName(d) => d.as_ref().into(),
+            Self::IpAddress(i) => {
+                let ip: std::net::IpAddr = (*i).into();
```

Comment:
Can we just make this `std::net::IpAddr::from(*i).to_string().into()`?

Reply:
Good call, thanks. Fixed up.